spec/file_reconstruction.md

```diff
@@ -0,0 +1,95 @@
+# File Reconstruction
+
+After a file is mapped into a series of chunks, and those chunks are tracked by xorbs, we can define the file reconstruction or the recipe to re-materialize the file.
+
+For every chunk of the file we need to know which xorb this chunk is in and its index (starting at 0) in that xorb and list out all the chunks in this way to re-create the file.
+
+Now since xorbs contain multiple chunks, we can condense the listing of each range of chunks that are contiguous in the same xorb together and this map the listing of chunks that form the file to a series of sub-ranges of chunks within xorbs.
```

Comment:
We might want to indicate that this contiguous range of chunks within a xorb is referred to as a **term**